packages/auto-tls/src/auto-tls.ts

```diff
@@ -77,6 +79,8 @@ export class AutoTLS implements AutoTLSInterface {
     const base36EncodedPeer = base36.encode(this.components.peerId.toCID().bytes)
     this.domain = `${base36EncodedPeer}.${this.forgeDomain}`
     this.email = `${base36EncodedPeer}@${this.forgeDomain}`
+    this.userAgent = init.userAgent ?? `${this.components.nodeInfo.name}/${this.components.nodeInfo.version} ${process.release.name}/${process.version.replaceAll('v', '')}`
```

Comment:
Could the user agent string could contain unexpected characters from `nodeInfo.name` or `nodeInfo.version`?

Reply:
We pass the user agent string to axios or undici, if there are characters that trip them up then I think the bug is with those modules.

What sort of characters are you thinking of?